setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 4.0.1
+current_version = 4.0.2
```

Comment:
And here

Reply:
Instead of manually changing the version, you can use the `bumpversion` tool to keep version numbers synced everywhere they are referecenced in the repo. The Makefile has commands to help this process. For example when you run `make bumpversion-minor` it will find the current version in the `setup.cfg` which in this case should be `4.0.1`. Then it will determine that applying a patch means the new version will be incremented to `4.1.0`. It looks at all the files and patterns specified in the `setup.cfg` to replace told version with the new version. In our case, it will update the version in the `setup.py` and and` __init__.py`. It then replaces the current version with `4.1.0` and creates a new git commit with a tag as a reference for the exact commit of the release.

It might be easier to wait until after the PR has been merged and then performing the version bump directly on master branch before releasing, but I'm not 100% sure.

I think this might also mean adjusting how we use the changelog. What if we keep a section at the top that accumulates a list of changes from all the PRs, and when we decide to do a release we make a new section with the incremented version number and the date and move all the changes into the versioned section? Then the accumulating process would start again

Reply:
For the sake of simplicity and getting this fix out, I've made the change manually this time, but will use the makefile with future PRs